spec/helpers/http_mock.lua

```diff
@@ -25,6 +25,32 @@ for _, module in ipairs(modules) do
   end
 end
 
+-- get a session from the logs with a timeout
+-- throws error if no request is recieved within the timeout
+-- @treturn table the session
+function http_mock:get_session()
```

Comment:
Are any of these new methods intended to be used somewhere in the PR?

Reply:
nvm, I see this is a backport of changes made in https://github.com/Kong/kong-ee/pull/6868